Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1311,8 +1313,8 @@ def remove_tag(demisto_args: dict, is_attribute=False):
     Function will remove tag to given UUID of event or attribute.
     is_attribute (bool): if the given UUID is an attribute's one. Otherwise it's event's.
     """
-    uuid = demisto_args.get('uuid')
-    tag = demisto_args.get('tag')
+    uuid = demisto_args['uuid']
```

Comment:
are we sure about changing this?

Reply:
in this particular case, `uuid` is `required`